server/src/main/java/org/apache/iotdb/db/mpp/common/header/ColumnHeaderConstant.java

```diff
@@ -200,7 +202,7 @@ public class ColumnHeaderConstant {
           new ColumnHeader(COLUMN_SERIES_SLOTS, TSDataType.INT64),
           new ColumnHeader(COLUMN_TIME_SLOTS, TSDataType.INT64),
           new ColumnHeader(COLUMN_DATANODE_ID, TSDataType.INT32),
-          new ColumnHeader(COLUMN_HOST, TSDataType.TEXT),
+          new ColumnHeader(COLUMN_RPC_PORT, TSDataType.TEXT),
           new ColumnHeader(COLUMN_PORT, TSDataType.INT32),
```

Comment:
COLUMN_RPC_PORT and COLUMN_PORT? Is there exist any failure?

Reply:
i will remove is and add test screenshot